exercises/isogram/test/isogram_test.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:test/test.dart';
+import 'package:isogram/isogram.dart';
+
+void main() {
+  final isogram = new Isogram();
+
+  group('Isogram', () {
+    group('Check if the given string is an isogram', () {
+      test("empty string", () {
+        final bool result = isogram.isIsogram("");
+        expect(result, equals(true));
+      }, skip: false);
```

Comment:
All other tests after this should be `skip: true`

Reply:
Skipped all but the first.